### PR TITLE
Add stub "Integrations" page; adjust nav (take 2)

### PR DESCRIPTION
**Pull request by aqw**
Supersedes #57 (after my delightful accidental push to master....)

- adds a stub Integrations page and adds to the nav
- tweaks the nav a bit. It's still clunky, but this fixes some bugs in the phablet/tablet resolution ranges, and gives us enough room for the integrations link to actually fit.
  - More nav work to come in the future, likely with remove/reworking the largely meaningless banner.
- removes the "Get DataLad" button from the landing page

Works towards #49

### CODE REVIEW

File: content/pages/index.html [hunk 1]
```diff
@@ -6,9 +6,6 @@
   <meta name="url" content="/index.html" />
 </head>
 <body>
-  <div class='download'>
-    <p><a href='/get_datalad.html'>Get DataLad</a></p>
-  </div>
```
Comment (yarikoptic) [inline content/pages/index.html]: Is there another obvious way to get there?

Comment (aqw) [reply content/pages/index.html]: Yes. The "Get DataLad" link in the navbar.

**Comment by aqw** [comment]
Sorry; figured you'd just pull and do a ``make devserver``

image

File: content/pages/index.html (hunk 1 above)
Comment (yarikoptic) [reply content/pages/index.html]: Cool, Thanks

**Comment by aqw** [comment]
> Replace lipsum with facts

I object to that characterization. "The only thing worse than yellow snow is green snow" is one of the great truisms of life. ;-)

**Comment by mih** [comment]
Thx, going in!

**Pull request merged**